release-notes/opensearch-knn.release-notes-1.0.0.0.md

```diff
@@ -1,6 +1,15 @@
 ## Version 1.0.0.0 Release Notes
 
 Compatible with OpenSearch 1.0.0
+
+### Features
+
+* Refactor interface to support method configuration in field mapping ([#20](https://github.com/opensearch-project/k-NN/pull/20))
+
```

Comment:
I think you have to manually add additional space types supported. PR was merged in ODFE and then migrated to OpenSearch

Reply:
@jmazanec15 was that part of beta? if so we don't have to add it here.

Reply:
It was, but I think it should be included here too

Reply:
Any feature after  odfe1.13.2 should be added